### DISCUSSION TITLE
docs/design

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#21] [General]
# docs/design

设计思想是 GoFrame 框架的灵魂，同时对于使用者来讲，是不可或缺的内功心法。

https://goframe.org/docs/design

**Comment by oldme-git** [comment]
好！

**Comment by UncleChair** [comment]
哇，新官网！

**Comment by wulie** [comment]
新手小白:有一个问题咨询一下， 咱们这个新官网感觉很不错，想问下是使用的什么工具搭建的， 开源的还是自己开发的

**Reply by oldme-git** [reply]
docusaurus

**Reply by wulie** [reply]
谢谢
